db/anonymise/rules.rb

```diff
@@ -91,7 +91,11 @@
   cfe_submissions: {
     cfe_result: nil,
   },
-  chances_of_successes: {},
+  chances_of_successes: {
+    success_prospect_details: lambda do |original_value, row_context|
+      row_context[:field_1_val] = Faker::Lorem.paragraph(sentence_count: 2) unless original_value.eql?('\N')
+    end,
```

Comment:
Oooh, is this to handle the last column problem?

Reply:
sadly not. its just a weird way of setting a fields value conditionally - :field_1_val in this context is the first field _mentioned_ in the hash for the chances_of_successes table COPY statements. If I added another column/field to be anonymised above it i would have to amend this to  `field_1_val`.

I don't like it but it seems to be the only way to not fill a field that would have been null